docs/non-normative/groups/system/design-philosophy.md

```diff
@@ -0,0 +1,94 @@
+# **System Semantic Conventions: Instrumentation Design Philosophy**
```

Comment:
I wasn't sure an OTEP was the right place for this. [This section for the oteps README](https://github.com/open-telemetry/oteps?tab=readme-ov-file#what-changes-require-an-otep) doesn't cover semantic conventions stuff super well, but since this document only nominally affects what is owned by System Semantic Conventions WG it seems to fall under `only affects one implementation`. (I am interpreting the doc pretty liberally though)

I had brought up before that I wasn't sure where this type of doc should live, and `non-normative` was what we thought seemed like the best place. I'm open to putting it in a better place, but I wonder if this is too centralized to one group to be a full OTEP.

Reply:
Yeah I also think this would be great as an OTEP. We did several in past for stuff like messaging so there is precedent for semconv in there, but I'm also not opposed in having it here. 

One thing we need to verify is about the website. If such documents should be included there and if so, where.

Reply:
Let's keep it here - it's always easy to move if necessary. Spec now have OTEPs inside the repo, let's keep `non-normative` as our local alias for semconv-scoped oteps.